tests/test_rebase_command.rs

```diff
@@ -20,15 +20,14 @@ pub mod common;
 
 fn create_commit(test_env: &TestEnvironment, repo_path: &Path, name: &str, parents: &[&str]) {
     if parents.is_empty() {
-        test_env.jj_cmd_success(repo_path, &["new", "root"]);
+        test_env.jj_cmd_success(repo_path, &["new", "root", "-m", name]);
```

Comment:
Was this intended to be in the prior commit?

Reply:
No, this ("test_rebase_command: avoid unnecessary `close` command") is the right commit, actually. The previous commit ("test_rebase_command: avoid an unnecessary special case") was about using `new` instead of `checkout` since it works with any number of commits. The removal of the `close` command in this commit meant that nothing was setting the description anymore, which made some tests fail, so I had to add that to this call.